### DISCUSSION TITLE
Toggle

### DISCUSSION THREAD

**Discussion by pedrolaxe** [strapi/strapi#7732] [Legacy Topics]
Hi Guys, how i create one field to enable another fields, like toggle?

**Comment by MattieBelt** [comment]
Hey @pedrolaxe, there isn't anything like an `dependsOn` option for fields. So there is no 'toggle' available for the admin content manager by default. In the community Slack it was recently appointed as a feature-request, but as far as I know nothing is offically planned to develop this.

For a sort like solution you can modify your create/update controllers to execute some custom validation on the created/updated entry. Check out [the docs](url) to extend your controller.

**Comment by derrickmehaffy** [comment]
Mattie is correct, we don't currently have a fields `depends` system or commonly referred to as a Parent/Child field relationship. It's a feature that has been suggested in the past using a car Make and Model as an example

(I should be able to select Make = Ford and Model = Corvette)

We have not currently implemented this, nor do we have any guides on doing so, but you are of course welcome to custom your own project to handle that case and should you desire to commit your solution in a pull request to us, I suggest you first file an RFC: https://github.com/strapi/rfcs
